twisted/conch/test/test_cftp.py

```diff
@@ -555,9 +555,8 @@ def makeFile(self, path=None, content=b''):
         """
         if path is None:
             path = self.mktemp()
-        file = open(path, 'w')
-        file.write(content)
-        file.close()
+        with open(path, 'w') as file:
```

Comment:
Filepath (path).setContent (content) only accepts binary input, so in that case "content" would need to be a binary string.  Be careful in going in this direction because Python3 is more fussy about this.  The patch as submitted does not have this issue and is probably good enough.